src/Fetchers/TMSFetcher/TMSFetcher.ts

```diff
@@ -139,9 +148,39 @@ export class TMSFetcher extends Fetcher {
      */
     descriptionToAppend(competition: Competition, match: Match,
                         html: boolean): string[] {
-
         const lines: string[] = [];
 
+        // Add officials if present
+        const officials = match.getOfficials();
+        if (officials.length > 0) {
+            lines.push("");
+            lines.push("Match Officials:");
+
+            // Define role order based on table columns
+            const roleOrder = [
+                "Umpire",
+                "Reserve/Video",
+                "Scoring/Timing",
+                "Technical Officer"
+            ];
+
+            // Sort officials by predefined role order
+            const sortedOfficials = [...officials].sort((a, b) => {
+                const indexA = roleOrder.indexOf(a.role);
```

Comment:
This needs a `'?? 0'` in case we encounter a role not in the fixed order list.

Reply:
Done